Rajashree/ass1q1.html

```diff
@@ -0,0 +1,11 @@
+<html>
+<head>
+<title>  Que1:Write a HTML prog to get the following output</title>
+</head>
+<body>
+<h1> HTML</h1>
+<h2> Introduction</h2>
+<h3><pre> HTML is a standard markup language for Web pages</h3></pre>
```

Comment:
@Rajashree07 tag which is opened first should be closed first. So, </pre> should be before </h3>
check this in other files as well